The ext-geth tests are failing due to a build issue of op-geth due to a go mod dependency issues, see "view details" above;
```
make[1]: Entering directory '/root/project/op-e2e/external_geth'
go build -o shim .
go build -o op-geth "github.com/ethereum/go-ethereum/cmd/geth"
# github.com/golang/protobuf/protoc-gen-go/descriptor
/go/pkg/mod/github.com/golang/protobuf@v1.5.3/protoc-gen-go/descriptor/descriptor.pb.go:106:61: undefined: descriptorpb.Default_FileOptions_PhpGenericServices
make[1]: *** [Makefile:4: op-geth] Error 1
make[1]: Leaving directory '/root/project/op-e2e/external_geth'
make: *** [Makefile:15: test-external-geth] Error 2
```
Not sure what changed here with the introduction of asterisc, but that's what's blocking the merge queue. Maybe rebasing the PR helps.

--- a/bedrock-devnet/devnet/__init__.py
+++ b/bedrock-devnet/devnet/__init__.py
@@ -141,7 +141,7 @@ def devnet_l1_genesis(paths):
 
     fqn = 'scripts/Deploy.s.sol:Deploy'
     run_command([
-        'forge', 'script', '--chain-id', '900', fqn, "--sig", "runWithStateDump()"
+        'forge', 'script', '--chain-id', '900', fqn, "--sig", "runWithStateDump()", "--private-key", "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
     ], env={}, cwd=paths.contracts_bedrock_dir)
 
     forge_dump = read_json(paths.forge_dump_path)

--- a/bedrock-devnet/devnet/__init__.py
+++ b/bedrock-devnet/devnet/__init__.py
@@ -140,6 +140,7 @@ def devnet_l1_genesis(paths):
     init_devnet_l1_deploy_config(paths)
 
     fqn = 'scripts/Deploy.s.sol:Deploy'
+    # Use foundry pre-funded account #1 for the deployer
     run_command([
         'forge', 'script', '--chain-id', '900', fqn, "--sig", "runWithStateDump()", "--private-key", "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
     ], env={}, cwd=paths.contracts_bedrock_dir)

--- a/op-program/Makefile
+++ b/op-program/Makefile
@@ -26,6 +26,9 @@ op-program-client-mips:
 	# verify output with: readelf -h bin/op-program-client.elf
 	# result is mips32, big endian, R3000
 
+op-program-client-riscv:
+	env GO111MODULE=on GOOS=linux GOARCH=riscv64 go build -v -gcflags="all=-d=softfloat" $(LDFLAGS) -o ./bin/op-program-client-riscv.elf ./client/cmd/main.go
+
 reproducible-prestate:
 	@docker build --output ./bin/ --progress plain -f Dockerfile.repro ../
 	@echo "Absolute prestate hash:"
@@ -74,6 +77,7 @@ verify-compat: verify-sepolia-delta verify-sepolia-ecotone verify-mainnet-genesi
 	op-program-host \
 	op-program-client \
 	op-program-client-mips \
+	op-program-client-riscv \
 	clean \
 	test \
 	capture-goerli-verify \

